app/services/record_declarations/event_factory.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+module RecordDeclarations
+  class EventFactory
+    class << self
+      def call(event)
+        event_namespace_for_event(event)
+      end
+
+    private
+
+      def event_namespace_for_event(event)
+        event_identifiers[event.underscore.intern].to_s
+      end
+
+      def event_identifiers
+        started_identifiers.merge(retained_identifiers)
+      end
+
+      def started_identifiers
```

Comment:
This highlights to me that we should *not* be conflating "started" and "completed" as they are only coincidentally the same. This conflation is visible externally in https://github.com/DFE-Digital/early-careers-framework/blob/13bebc716998e5ab8942ac721d92ef5e7b5ccdb5/swagger/v1/component_schemas/ECFParticipantStartedDeclaration.yml
We should follow up with further work to disentangle started and completed across the whole stack.